lib/charms/opensearch/v0/opensearch_tls.py

```diff
@@ -423,16 +423,21 @@ def get_unit_certificates(self) -> Dict[CertType, str]:
 
     def _create_keystore_pwd_if_not_exists(self, scope: Scope, cert_type: CertType, alias: str):
         """Create passwords for the key stores if not already created."""
-        keystore_pwd = None
+        store_pwd = None
+        if alias == "ca":
+            store_type = "truststore"
+        else:
+            store_type = "keystore"
```

Comment:
nit for conciseness:
```suggestion
        store_type = "truststore" if alias == "ca" else "keystore"
```